src/custom-users-for-firebase/user-firebase.js

```diff
@@ -0,0 +1,53 @@
+export const users = [
```

Comment:
that is not something that should be present in git
never push any credentials or login info